README.md

```diff
@@ -66,3 +66,19 @@ Material for all KOSS workshops in a central location.
 │   ...
 └── README.md
 ```
+
+## [Workshop Wise Index](./index.md)
+
+1. [Golang Workshop](./index.md#golang-workshop)
```

Comment:
I think a link would suffice, maintaining two ToCs seems unnecessary and it will stop happening after some point :)

Reply:
Also, can you add a section to the README that explains that this file also has to be maintained?

Reply:
done